rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -72,7 +72,8 @@
         "fre",
         "eng",
         "ita",
-        "ger"
```

Comment:
Here is a lot of empty string for description in this file. Can we remove these?

Reply:
ok